pastel/client.go

```diff
@@ -230,7 +230,7 @@ func (client *client) RegTicket(ctx context.Context, regTxid string) (RegTicket,
 func (client *client) RegTickets(ctx context.Context) (RegTickets, error) {
 	tickets := RegTickets{}
 
-	if err := client.callFor(ctx, &tickets, "tickets", "list", "art"); err != nil {
+	if err := client.callFor(ctx, &tickets, "tickets", "list", "nft"); err != nil {
```

Comment:
Please check again with ayk.
Your changes seem correct - but I don't think https://github.com/pastelnetwork/pastel/pull/64/files corect "NFT" to "nft" for tickets list command.